metadata_service/proxy/neptune_proxy.py

```diff
@@ -65,6 +65,10 @@ def __init__(self, *, host: str, port: Optional[int] = None, user: str = None,
         if port is not None:
             raise NotImplementedError(f'port is not allowed! port={port}')
 
+        if client_kwargs.get('ignore_neptune_shard', False):
+            _reset_for_testing_only()
```

Comment:
@friendtocephalopods I know this is a private method but is this okay to use here?

Reply:
Hmm we shouldn't need to do do this here. If we'd like some os-level flag to not shard neptune in tests, let's add it to https://github.com/amundsen-io/amundsengremlin/blob/master/amundsen_gremlin/test_and_development_shard.py#L17 and just return None. Then we won't need this stuff propagating to clients of the library.